tests/providers/gcp/gcp_provider_test.py

```diff
@@ -817,6 +803,32 @@ def test_test_connection_with_exception(self):
             assert e.type == GCPTestConnectionError
             assert "Test exception" in e.value.args[0]
 
+    def test_test_connection_with_exception_service_account_key(self):
+        with patch(
+            "prowler.providers.gcp.gcp_provider.GcpProvider.setup_session",
+            side_effect=Exception("Test exception"),
+        ):
+            with pytest.raises(Exception) as e:
+                GcpProvider.test_connection(
+                    service_account_key={"test": "key"},
+                )
+            assert e.type == GCPTestConnectionError
+            assert "Test exception" in e.value.args[0]
+
+    def test_validate_static_arguments(self):
```

Comment:
Can you restore this to their original location?

Reply:
Done